Resources/config/pdf.xml

```diff
@@ -20,9 +20,9 @@
 	</parameters>
 
     <services>
-    	<service id="ps_pdf.facade" class="PHPPdf\Core\Facade" factory-service="ps_pdf.facade_builder" factory-method="build" />
+    	<service id="ps_pdf.facade" class="PHPPdf\Core\Facade" />
     	
-    	<service id="ps_pdf.facade_builder" class="PHPPdf\Core\FacadeBuilder" factory-class="PHPPdf\Core\FacadeBuilder" factory-method="create">
+    	<service id="ps_pdf.facade_builder" class="PHPPdf\Core\FacadeBuilder">
```

Comment:
A BC comment with Symfony version would be better to know when we can go back to simple configuration file.